src/Components/FormField.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function FormField(props){
+
+       const [state, setState] = props.state;  
+
```

Comment:
It's not mandatory but you could have written your `handleChange` function as assigned to a const. 

Personally, when I'm working with a functional component I try to only have the component itself as a function. You don't have to adopt this it's just something to consider.

Also you could have destructuring for `e.target`.
```
const handleChange = (e) =>{
        const { name, value } = e.target;
        setState({...state,[name]: value});
    }